### PR TITLE
Base application

### PR DESCRIPTION
**Pull request by aamir-asaram**
## Changes made

- [x] Setup webpack 
- [x] Add basic structure of the todo list matching the template.
- [x] Added basic hover effects

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,178 @@
...
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
```
Comment (V-Blaze) [inline README.md]: - [OPTIONAL] I'll suggest you remove this comment as it's just a place holder from the readme template. :+1: :100:

File: README.md [hunk 2]
```diff
@@ -1 +1,178 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
```
Comment (V-Blaze) [inline README.md]: - [OPTIONAL] I'll suggest you also remove this comment as it's just a placeholder from the readme template. :+1: :100:

File: README.md [hunk 3]
```diff
@@ -1 +1,178 @@
...
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for providing the resources necessary for me to complete this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
```
Comment (V-Blaze) [inline README.md]: - [OPTIONAL] I'll suggest you remove this FAQ, heading also as you don't have any content in it. Leaving an unused header makes your readme file look a bit unprofessional. :smile: :wave:

**Review by V-Blaze — APPROVED** [review]
Hi @aamir-asaram  

# Approved 🍏 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
green-lantern-ok

## Highlights
- Webpack setup correctly✔✔
- Todo items displays properly on the UI :heavy_check_mark: 
- Clean DRY code :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (aamir-asaram) [reply README.md]: Thanks for picking that up @V-Blaze, I missed those completely :sweat_smile:

**Pull request merged**